pkg/controllers/cluster/cluster_controller.go

```diff
@@ -244,21 +230,19 @@ func (c *Controller) removeCluster(ctx context.Context, cluster *clusterv1alpha1
 		return controllerruntime.Result{}, err
 	} else if exist {
 		klog.Infof("Requeuing operation until the cluster(%s) execution space deleted", cluster.Name)
-		return controllerruntime.Result{RequeueAfter: c.ExecutionSpaceRetryFrequency}, nil
+		return controllerruntime.Result{RequeueAfter: c.ExecutionSpaceRemoveWaitingRetryFrequency}, nil
 	}
 
-	// delete the health data from the map explicitly after we removing the cluster.
+	// delete the health data from the map explicitly after we're removing the cluster.
 	c.clusterHealthMap.delete(cluster.Name)
 
 	// check if target cluster is removed from all bindings.
-	if c.EnableTaintManager {
-		if done, err := c.isTargetClusterRemoved(ctx, cluster); err != nil {
-			klog.ErrorS(err, "Failed to check whether target cluster is removed from bindings", "cluster", cluster.Name)
-			return controllerruntime.Result{}, err
-		} else if !done {
-			klog.InfoS("Terminating taint eviction process has not finished yet, will try again later", "cluster", cluster.Name)
-			return controllerruntime.Result{RequeueAfter: c.ClusterTaintEvictionRetryFrequency}, nil
-		}
+	if done, err := c.isTargetClusterRemoved(ctx, cluster); err != nil {
+		klog.ErrorS(err, "Failed to check whether target cluster is removed from bindings", "cluster", cluster.Name)
+		return controllerruntime.Result{}, err
+	} else if !done {
+		klog.InfoS("Terminating taint eviction process has not finished yet, will try again later", "cluster", cluster.Name)
```

Comment:
Should twist the logs as here does not rely on the eviction process anymore.

Reply:
Updated.